fdbserver/IPager.h

```diff
@@ -40,6 +40,45 @@ typedef uint32_t PhysicalPageID;
 typedef uint32_t QueueID;
 #define invalidQueueID std::numeric_limits<QueueID>::max()
 
+// Pager Events
+enum class PagerEvents { pagerCacheLookup = 0, pagerCacheHit, pagerCacheMiss, pagerWrite, MAXEVENTS };
+static const std::string PagerEventsCodes[] = { "Lookup", "Hit", "Miss", "Write" };
+// Reasons for page level events.
+enum class PagerEventReasons { pointRead = 0, rangeRead, rangePrefetch, commit, lazyClear, metaData, MAXEVENTREASONS };
+static const std::string PagerEventReasonsCodes[] = { "Get", "GetR", "GetRPF", "Commit", "LazyClr", "Meta" };
+
+static const int nonBtreeLevel = 0;
+static const int rootLevel = 1;
+static const std::pair<PagerEvents, PagerEventReasons> possibleEventReasonPairs[] = {
+	{ PagerEvents::pagerCacheLookup, PagerEventReasons::pointRead },
+	{ PagerEvents::pagerCacheLookup, PagerEventReasons::rangeRead },
+	{ PagerEvents::pagerCacheLookup, PagerEventReasons::lazyClear },
+	{ PagerEvents::pagerCacheLookup, PagerEventReasons::metaData },
+	{ PagerEvents::pagerCacheLookup, PagerEventReasons::commit },
+	{ PagerEvents::pagerCacheHit, PagerEventReasons::pointRead },
```

Comment:
Page reads to support `commit` happen at a BTree level and will either be cache hits or misses.